src/index.js

```diff
@@ -1,13 +1,59 @@
-// This is the JavaScript entry file - your code begins here
-// Do not delete or rename this file ********
-
-// An example of how you tell webpack to use a CSS (SCSS) file
+//****========= IMPORTED FILES =========****}}>
 import "./css/base.scss";
 import "./css/index.scss";
+import "./images/bed.svg";
+import "./images/bidet.svg";
+import "./images/desk-bell.svg";
+import "./images/hotel.svg";
+import "./images/search.svg";
```

Comment:
Totally missed that there were image files to import. Have you got these working on the dom?

Reply:
Echoing Jeff's question

Reply:
I just added ones I wanted to use for my design from flaticon.com!